README.md

```diff
@@ -193,7 +192,7 @@ We are planning to make descisions and reach consensus during specific stages of
 
 We want to on core questions of this proposal as a Stage 2 prerequisite, including:
```

Comment:
I guess we should say, "We have achieved consensus on the following core decisions as part of Stage 2, including:"